config/openhpc.yml

```diff
@@ -106,7 +108,7 @@ gluster_mnt: "/mnt/{{ gluster_cluster_volume_name }}"
 # Config for Ceph mount
 ceph_mount_share_name: HomeDirs
 ceph_mount_path: /alaska
-ceph_mount_fuse: true
+ceph_mount_fuse: false
```

Comment:
Seems a bit unrelated

Reply:
Its related in the sense that updating to latest packages allows us to use kernel rather than fuse mount

Reply:
Does that mean it's now necessary to run the upgrade playbook? Is that documented?

Reply:
Its part of the openhpc.yml playbook so should take place automatically.

Reply:
Right you are.